app/engine/hooks/transactions/useAccountTransactions.ts

```diff
@@ -36,13 +34,13 @@ export function parseStoredMetadata(metadata: StoredContractMetadata): ContractM
 }
 
 
-export function useAccountTransactions(client: TonClient4, account: string, refetchOnMount: boolean = false): {
+export function useAccountTransactions(account: string, refetchOnMount: boolean = false): {
```

Comment:
let's do this in object for better readability, like: { refetchOnMount: true }

Reply:
Sure, you are right ✅